client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts

```diff
@@ -54,18 +64,27 @@ export class ActiveWildfireMapComponent implements OnInit, OnChanges {
         private wfMapService: WFMapService,
     ) {
         this.incidentsServiceUrl = this.appConfig.getConfig().rest['newsLocal'];
+        this.placeData = new PlaceData();
+        
         // console.log(this.incidentsServiceUrl)
+
+        this.placeData.setResultHandler((result) => {
+            this.filteredOptions = result.roads;
+        });
+
+        this.searchByLocationControl.valueChanges.subscribe((val:string)=>{
+            if(!val) this.filteredOptions= [];
+            if(val.length > 2) {
+                this.placeData.findRoad(val);
+            }
+        });
     }
 
     ngOnInit() {
         this.showAccordion = true;
         this.appConfig.configEmitter.subscribe((config) => {
             const mapConfig = [];
-            // this.checkMapServiceStatus()
-            // .then( ( mapServiceStatus ) => {
-            // console.log(mapServiceStatus)
-            // this.wfnewsMapService.mapServiceStatus = mapServiceStatus
-
+            
             this.mapConfigService.getMapConfig() // this.applicationConfig.device )
                 // .then((config) => {
                 //     mapConfig.push(config)
```

Comment:
If it isn't used, we can remove the commented out stuff

Reply:
Sure, I wasn't sure to delete the comments, but I'll go ahead and start to remove those.